server/routes/api.js

```diff
@@ -64,6 +64,22 @@ router.get('/Students', (req, res) => {
     });
 });
 
+// Get Students ID
+router.get('/Students/:id', (req, res) => {
+    connection((db) => {
+        db.collection('Students')
+              .find({"username":req.params.id})
```

Comment:
Ok so I have two comments/concerns.

1) I think it would be better to query based off of _id.  While we will probably need a username based query at some point it is probably better to just use `_id` here instead of `username`.  So instead of `/api/students/ltcarbonell` it would be `/api/students/31469461483642`.  I know this is slightly different from what the tickets say, but I feel like this would be a better option.  Once we implement logging in is when we can worry about usernames/password and whatnot

2) The mongo query here is a find which means it returns an array of students as opposed to a findOne which would only return the exact match.  Now technically since we are querying on a unique id there should be only one results, however I'm not sure performance wise if findOne is better to use.  Maybe look into this and if there isn't a significant difference between the two then I am okay with it the way it is.

Reply:
1.I can revise the code so that it could be based on id instead of username.
2.I think every object has different id, thus, if it is based on id. I think findOne could have the same result as find based on id.

Reply:
Ok.  Both of those are fine with me.